ibis/expr/api.py

```diff
@@ -717,6 +717,7 @@ def _time_from_deferred(value: Deferred) -> Deferred:
 def interval(
     value: int | datetime.timedelta | None = None,
     unit: str = 's',
+    *,
```

Comment:
It might be good to note this as a breaking change. Up to you.

Reply:
Calling `interval(1, 's', 2)` wouldn't have worked before either. Changing the signature just makes the lack of this working more visibly explicit.